build.xml

```diff
@@ -27,6 +27,7 @@
 
   <!-- Classpath for building and running tests -->
   <path id="classpath-tests">
+    <pathelement location="src/test/resources"/>
```

Comment:
I have added this line to include simplelogger.properties into classpath. I thought this should be the proper way using ant. As I am not expert of Any I was not able to show properties file to slf4j is there other way to this?